im-palette.R

```diff
@@ -0,0 +1,176 @@
+  `MOJN BLUE` = "#6ECEF4",
+  `MOJN BLUE D` = "#2C5261",
+  `MOJN WHITE L` = "#FFFFFF",
+  `MOJN WHITE` = "#DDDDDD",
+  `MOJN WHITE D` = "#BBBBBB",
+  `MOJN GRAY L` = "#D0D0D0",
+  `MOJN GRAY` = "#A1A1A1",
+  `MOJN GRAY D` = "#404040",
+  `MOJN BLACK L` = "#808080",
+  `MOJN BLACK` = "#222222",
+  `MOJN BLACK D` = "#000000",
+  `NPS BLUE` = "#BAE5F9",
+  `NPS ORANGE` = "#C56C39",
+  `NPS BROWN` = "#6F4930",
+  `NPS GREEN` = "#56903A",
+  `NPS BROWN D` = "#213A1B",
+  `DULL PINK` = "#6F5159",
+  `WARM GRAY` = "#ACA9A2",
+  `SPRING BLUE` = "#5585A9",
+  `AQUA` = "#9BDEEF",
+  `RICH BROWN` = "#385424",
+  `EARTH GREEN` = "#2D782C",
+  `PALE BROWN` = "#FAC9A1",
+  `TAN BROWN` = "#FDAA42",
+  `SIMPLE BROWN` = "#E95B0F",
+  `DEEP AQUA` = "#4AB0F2",
+  `MARINE BLUE` = "#013040",
+  `BRICK RED` = "#B42C02",
+  `SEA FOAM` = "#CCEBF0",
+  `DUSTY BROWN` = "#C0AA81",
+  `KHAKI` = "#AFC871",
+  `DARK AQUA` = "#3076B1",
+  `DULL BLUE` = "#234088",
+  `DULL BROWN` = "#32332D",
+  `DULL RED` = "#BF2E0F",
+  `BROWN` = "#BB853F",
+  `COOL BROWN` = "#C7BEAF",
+  `BLUE GRAY` = "#414550",
+  `DULL AQUA` = "#7FA2C0",
+  `FLAT AQUA` = "#57C9DD",
+  `MAGENTA` = "#B45090",
+  `PURPLE` = "#C69CD0",
+  `RICH GREEN` = "#6B8637",
+  `GREEN` = "#59AB47",
+  `BLUE GREEN` = "#81ADD2",
+  `DULL YELLOW` = "#E1E100",
+  `PALE AQUA` = "#DEE4F0",
+  `LIGHT BLUE` = "#679CD0",
+  `FLAT BLUE` = "#596980",
+  `OCEAN BLUE` = "#003C70",
+  `FLAT BROWN` = "#80815A",
+  `SILT BROWN` = "#806645",
+  `FADED BLUE` = "#6899BA",
+  `LIGHT ROSE` = "#DDCCBA",
+  `ROSE` = "#CC7667",
+  `FAR SHADOW` = "#432233",
+  `BLUE SLATE` = "#8C8F9E",
+  `DEEP BEIGE` = "#B8B397",
+  `WARM BEIGE` = "#F6D2B8",
+  `BEIGE` = "#F6BA90",
+  `STOPSIGN RED` = "#FF6E42",
+  `MUDDY BROWN` = "#9B5B4F",
+  `RED CLAY` = "#8F5F39",
+  `DEEP CAVE` = "#303129",
+  `BRIGHT BEIGE` = "#FFCCB1",
+  `SKY BLUE` = "#99CFFE",
+  `GREEN STONE` = "#7FAA9F",
+  `DRY GREEN` = "#A6B683",
+  `SHRUB BARK` = "#869463",
+  `PINE NEEDLE` = "#395137",
+  `BROWN GRAY` = "#867A6E",
+  `WET BARK` = "#587040",
+  `FOGGY LEAF` = "#99C383",
+  `FOREST MIST` = "#9DDFDE",
+  `SHADED MIST` = "#4D6160",
+  `FOREST STREAM` = "#303944",
+  `SANDY HILL` = "#FCEAA0",
+  `RAINCLOUD` = "#A6B4BF",
+  `DIRT` = "#928C2E",
+  `BLUE RIVER` = "#3B7492",
+  `RED BROWN` = "#A56621",
+  `DEEP RIVER` = "#475965",
+  `DARK LEAF` = "#548F01",
+  `FOREST BROWN` = "#B5B016",
+  `REDWOOD` = "#8B2F00",
+  `CONIFER` = "#40663F",
+  `DRY MEADOW` = "#DDE02B",
+  `DAMP MOSS` = "#848B64",
+  `CLOUD` = "#EBECF0",
+  `RIVER` = "#5B659F",
+  `LAKE` = "#01479D",
+  `DRY SHRUB` = "#D58A60",
+  `DRY HAY` = "#FFC283",
+  `WET SOIL` = "#5A6441",
+  `ORANGE` = "#F7941D"
+)
+
+# NOTE: Using Roxygen formatting for comment blocks so that it will be easy to make this into a package later
+
+#' Function to extract colors as hex codes
+#'
+#' @param ... Character names of colors
+#'
+im_cols <- function(...) {
+  cols <- c(...)
+  
+  if (is.null(cols)) {
+    return(im_colors)
```

Comment:
in the `im_cols()` function, returning all the colors if the input is `NULL` is handy. Do you want to add the same functionality to the `im_pal()`  function?

Reply:
I'm not opposed, but I was thinking it might be more helpful to default to a specific palette (with standard NPS colors, maybe) instead of trying to interpolate all the colors. How about we make a decision on that once we've narrowed down the color list? If we narrow it down enough I could see it making sense to just default to all colors. Regardless, I definitely agree that we want a sensible default when input is null.